user-interface/src/main/java/life/qbic/datamanager/views/projects/project/experiments/ExperimentInformationMain.java

```diff
@@ -49,44 +40,26 @@ public class ExperimentInformationMain extends MainComponent implements BeforeEn
   private static final Logger log = logger(ExperimentInformationMain.class);
   public static final String EXPERIMENT_ID_ROUTE_PARAMETER = "experimentId";
   public static final String PROJECT_ID_ROUTE_PARAMETER = "projectId";
-  private final ProjectNavigationBarComponent projectNavigationBarComponent;
   private final ExperimentContentComponent experimentContentComponent;
   private final ExperimentSupportComponent experimentSupportComponent;
-  private final transient ProjectInformationService projectInformationService;
-  private final transient ExperimentInformationService experimentInformationService;
   private transient Context context;
 
-  public ExperimentInformationMain(
-      @Autowired ProjectNavigationBarComponent projectNavigationBarComponent,
-      @Autowired ExperimentContentComponent experimentContentComponent,
-      @Autowired ExperimentSupportComponent experimentSupportComponent,
-      @Autowired ProjectInformationService projectInformationService,
-      @Autowired ExperimentInformationService experimentInformationService) {
+  public ExperimentInformationMain(@Autowired ExperimentContentComponent experimentContentComponent,
+      @Autowired ExperimentSupportComponent experimentSupportComponent) {
     super(experimentContentComponent, experimentSupportComponent);
-    requireNonNull(projectNavigationBarComponent);
     requireNonNull(experimentSupportComponent);
     requireNonNull(experimentContentComponent);
-    requireNonNull(projectInformationService);
-    requireNonNull(experimentInformationService);
-    this.projectNavigationBarComponent = projectNavigationBarComponent;
     this.experimentContentComponent = experimentContentComponent;
     this.experimentSupportComponent = experimentSupportComponent;
-    this.projectInformationService = projectInformationService;
-    this.experimentInformationService = experimentInformationService;
-    layoutComponent();
+    addClassName("experiment");
     log.debug(String.format(
-        "New instance for ExperimentInformationMain (#%s) created with ProjectNavigationBar Component (#%s), ExperimentMain component (#%s) and ExperimentSupport component (#%s)",
```

Comment:
Please use variables to reference class names, since they are mutable and can change. `getClass().getName()` sth like that

Reply:
Oh good point! I took the liberty and adapted all class instantiation logs in this[commit](https://github.com/qbicsoftware/data-manager-app/pull/423/commits/fe3adef6dab78a40461cb086316da3a74a76f7f7) in a similiar way.